autonomi/README_PYTHON.md

```diff
@@ -97,6 +98,60 @@ The main interface to interact with the Autonomi network.
   - Calculate pointer storage cost
   - Returns cost in atto tokens
 
+#### Scratchpad
+
+Manage mutable encrypted data on the network.
+
+#### Scratchpad Class
```

Comment:
I'm not a fan of adding the API in the docs here, because it's really easy to get out of sync. But I do like the idea of putting some documentation in the README!